a3p-integration/proposals/z:acceptance/governance.test.js

```diff
@@ -26,17 +32,20 @@ test.serial(
     const instance = await readLatestHead(`published.agoricNames.instance`);
     const instances = Object.fromEntries(instance);
 
-    /** @type {any} */
-    const wallet = await readLatestHead(
-      `published.wallet.${governanceAddresses[0]}.current`,
-    );
+    const wallet =
+      /** @type {import('@agoric/smart-wallet/src/smartWallet.js').CurrentWalletRecord} */ (
+        await readLatestHead(
+          `published.wallet.${governanceAddresses[0]}.current`,
+        )
+      );
     const usedInvitations = wallet.offerToUsedInvitation;
 
     const charterInvitation = usedInvitations.find(
       v =>
         v[1].value[0].instance.getBoardId() ===
         instances.econCommitteeCharter.getBoardId(),
     );
+    assert(charterInvitation, 'missing charter invitation');
```

Comment:
why `assert` rather than `t.assert`?

Reply:
`t.assert` doesn't narrow (at least not in Ava 5)